schema/ListedLicense.xsd

```diff
@@ -29,46 +53,214 @@
 		<attribute name="deprecatedVersion" type="string"/>
 	</complexType>
 	<complexType name="LicenseType" mixed="true">
+		<annotation>
+			<documentation xml:lang="en">
+				<xhtml:p>
+					<xhtml:code>LicenseType</xhtml:code> is a license.  It does not represent exceptions, although the schema for <xhtml:code>ExceptionType</xhtml:code> is very similar.
+				</xhtml:p>
+			</documentation>
+		</annotation>
 		<all>
 			<element name="crossRefs" type="tns:crossRefsType" minOccurs="0" maxOccurs="1"/>
-			<element name="notes" type="tns:formattedFixedTextType" minOccurs="0" maxOccurs="1"/>
-			<element name="standardLicenseHeader" type="tns:formattedAltTextType" minOccurs="0" maxOccurs="1"/>
-			<element name="text" type="tns:formattedAltTextType" minOccurs="0" maxOccurs="1"/>
+			<element name="notes" type="tns:formattedFixedTextType" minOccurs="0" maxOccurs="1">
+				<annotation>
+					<documentation xml:lang="en">
+						<xhtml:p>
+							<xhtml:code>notes</xhtml:code> contains unstructured notes about the license.
+						</xhtml:p>
+					</documentation>
+				</annotation>
+			</element>
+			<element name="standardLicenseHeader" type="tns:formattedAltTextType" minOccurs="0" maxOccurs="1">
+				<annotation>
+					<documentation xml:lang="en">
+						<xhtml:p>
+							<xhtml:code>standardLicenseHeader</xhtml:code> is the license steward's recommended text for per-file license grants.
```

Comment:
IANAL, but from what I understand the term "grant" has specific legal meaning which may not be applicable to all situations where the license header is used.

Suggest changing "grant" to "notice" or "text".

Reply:
> … from what I understand the term "grant" has specific legal meaning which may not be applicable to all situations where the license header is used.

What's the distinction between “grant” and “notice”?  Maybe “Copyright 2018 me.  All rights reserved” would be a notice but not a grant?  That's fine, but since all of these are licenses, I expect all of our suggested headers will be license grants (although those license grants may also include copyright notices).